gcp-gke-core-services/service_account.tf

```diff
@@ -5,26 +5,26 @@ resource "kubernetes_namespace_v1" "md-core-services" {
   }
 }
 
-resource "kubernetes_service_account_v1" "cloud_provisioner" {
+resource "kubernetes_service_account_v1" "massdriver-cloud-provisioner" {
   metadata {
     name      = "massdriver-cloud-provisioner"
     labels    = var.md_metadata.default_tags
     namespace = kubernetes_namespace_v1.md-core-services.metadata.0.name
   }
 }
 
-resource "kubernetes_secret_v1" "main" {
+resource "kubernetes_secret_v1" "massdriver-cloud-provisioner_token" {
```

Comment:
whats up with the mix of dashes and underscores?

Reply:
Underscores should generally be used to separate words. The document some of the practices here: https://github.com/massdriver-cloud/massdriver-bundles/blob/main/BEST_PRACTICES.md#terraform-practices

However kubernetes doesn't allow underscores in names, so we have to use hyphens. Thus, the reason I used hyphens for the rest is the actual name of the resource in kubernetes is `massdriver-cloud-provisioner` with hyphens.